### PR TITLE
Feat navbar and filter

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
Added responsive navbar and filter in products page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,212 +1,247 @@
...
-
-                <p class="description-product">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae, iusto.</p>
-                <p class="price">₹ 549 <span class="price-offer">45% off</span></p>
-                <button class="hero-btn" >Add To Cart</button>
+    <div class="prominent-category">
+      <p class="category-header black-color">Prominent Category</p>
+      <div class="underline-header"></div>
+      <div class="category-container">
+        <div class="category-unit flex-center">
+          <img src="./assets/card_camera.jpg" alt="">
```
Comment (Kothalkarpriya) [inline index.html]: Do practise using / to close img tag as it is self closing tag

File: index.html [hunk 2]
```diff
@@ -1,212 +1,247 @@
...
+        <div class="category-unit flex-center">
+          <img src="./assets/card_img6.jpg" alt="">
+          <p class="category-content">Laptop</p>
+        </div>
+        <div class="category-unit flex-center">
+          <img src="./assets/watch.jpg" alt="">
+          <p class="category-content">Watch</p>
+        </div>
+        <div class="category-unit flex-center">
+          <img src="./assets/tablet.jpg" alt="">
```
Comment (Kothalkarpriya) [inline index.html]: everytime you used img tag u need to use / before closing

File: index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const collapeNavbarContainer = document.querySelector(".collape-navbar-container")
+// const overlayContainer = document.querySelector(".overlay-container")
+const menuLogo = document.querySelector(".menu-logo")
+const emptyBox = document.querySelector("#empty-box")
+
+// navbar_close_btn.addEventListener("click", () => {
+//     // collapeNavbarContainer.style.display = "none"
+//     // emptyBox.style.display = "none"
+//     collapeNavbarContainer.classList.add("hide-nav")
+//     collapeNavbarContainer.classList.remove("display-nav")
```
Comment (Kothalkarpriya) [inline index.js]: remove commented code if they are not required

File: index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+})
+
+menuLogo.addEventListener("click", () => {
+    collapeNavbarContainer.classList.remove("hide-nav")
+    collapeNavbarContainer.classList.add("display-nav")
+    emptyBox.classList.add("overlay-container")
+})
+
+
+
```
Comment (Kothalkarpriya) [inline index.js]: remove extra white spaces

File: productList.html [hunk 5]
```diff
@@ -77,6 +81,134 @@ <h2 class="main-title">
...
+              </div>
+            </div>
+    
+            <hr class="rg-line" />
+    
+            <div class="filter-inner-content">
+              <h3 class="mb-filter filter-content-title secondary-color">CATEGORY</h3>
+              <div class="mb-filter filter-item align-center">
+                <input type="checkbox">
+                <label for="">Category-1 </label>
```
Comment (Kothalkarpriya) [inline productList.html:160]: don't use empty attributes

File: productList.html [hunk 6]
```diff
@@ -77,6 +81,134 @@ <h2 class="main-title">
...
+                <input type="range" min="1" max="100" value="50">
+              </div>
+            </div>
+    
+            <hr class="rg-line" />
+    
+            <div class="filter-inner-content">
+              <h3 class="mb-filter filter-content-title secondary-color">CATEGORY</h3>
+              <div class="mb-filter filter-item align-center">
+                <input type="checkbox">
```
Comment (Kothalkarpriya) [inline productList.html:159]: you have used closing/tag for input before, so continue using it throughout the code

File: style/collapseNavbar.css [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  .abc {
+      background-color: transparent;
+      display: none;
+  }
+
+  /* .collape-navbar-container.hide-nav {
+    transform: translateX(0%);
+  }
+
+  .collape-navbar-container.display-nav {
```
Comment (Kothalkarpriya) [inline style/collapseNavbar.css:88]: remove comments if they are not needed

File: style/reset.css [hunk 8]
```diff
@@ -59,11 +59,22 @@ html {
   .product-list-container {
     grid-template-areas:
       "header header header header header header"
-      "product product product product product product";
+      "menu menu product product product product";
```
Comment (Kothalkarpriya) [inline style/reset.css:62]: you can minimize this line as "menu product product" it will also work

File: style/reset.css [hunk 9]
```diff
@@ -59,11 +59,22 @@ html {
...
   .header-main-content {
     display: none;
   }
+}
+
+@media only screen and (max-width: 500px) {
+  .product-list-container {
+    grid-template-areas:
+      "header"
+      "filter-mobile-layout"
```
Comment (Kothalkarpriya) [inline style/reset.css:74]: I think filter-mobile-layout is menu as you have used before, why did you change it? Just curious to know

File: wishlist.html [hunk 10]
```diff
@@ -21,68 +22,111 @@
...
           </div>
         </div>
       </div>
+  
+      <div id="empty-box"></div>
+  
+      <nav class="collape-navbar-container hide-nav">
+        <div class="header-navbar flex justify-between">
+          <h2>magni<span class="secondary-color highlight-letter-header">Z</span>ent</h2>
+          <button class="nav-close-btn"><span class="material-icons icon"> close </span></button>
```
Comment (Kothalkarpriya) [inline wishlist.html:91]: do use code formatting and different tags on a different line, it seems messy to find where the tag is being closed here

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into specified comments, and then you can merge

File: index.js [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+//     // console.log("closed")
+// })
+
+// menuLogo.addEventListener("click", () => {
+//     collapeNavbarContainer.classList.remove("hide-nav")
+//     collapeNavbarContainer.classList.add("display-nav")
+//     // emptyBox.classList.add("overlay-container")
+//     // console.log("open")
+// })
+
```
Comment (vaibhav18matere) [inline index.js]: please remove commented code (if you are not going to use it later)

File: productList.html [hunk 12]
```diff
@@ -77,6 +81,134 @@ <h2 class="main-title">
...
+                <label for="">3 Stars & above</label>
+              </div>
+              <div class="mb-filter filter-item align-center">
+                <input type="checkbox">
+                <label for="">2 Stars & above</label>
+              </div>
+              <div class="mb-filter filter-item align-center">
+                <input type="checkbox">
+                <label for="">1 Stars & above</label>
```
Comment (vaibhav18matere) [inline productList.html:190]: remove empty `for=""` or you can also use it for "checkbox" if you want

File: productList.html [hunk 13]
```diff
@@ -77,6 +81,134 @@ <h2 class="main-title">
...
+              </div>
+            </div>
+    
+            <hr class="rg-line" />
```
Comment (vaibhav18matere) [inline productList.html:194]: you could have use better and descriptive class name here

File: style/collapseNavbar.css [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+      display: none
+  }
+  .display-nav {
+      display: block;
+  }
+
+  .abc {
+      background-color: transparent;
+      display: none;
+  }
```
Comment (vaibhav18matere) [inline style/collapseNavbar.css]: use more descriptive class name here also. .`abc` is not so good

File: style/collapseNavbar.css [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+    transform: translateX(0%);
+  }
+
+  .collape-navbar-container.display-nav {
+    transform: translateX(100%);
+    transition: transform 0.5s ease;
+  } */
+}
+
+
```
Comment (vaibhav18matere) [inline style/collapseNavbar.css:94]: remove commented lines please

File: style/collapseNavbar.css [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+  .overlay-container {
+    display: block;
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```
Comment (vaibhav18matere) [inline style/collapseNavbar.css:45]: just use 0 once and it will set it to all directions, I think you don't need to write separately for every direction

**Review by vaibhav18matere — APPROVED** [review]
pretty clean and beautiful UI, here are few suggestions :

File: style/reset.css (hunk 9 above)
Comment (Vishruta-Patil) [reply style/reset.css:74]: > I think filter-mobile-layout is menu as you have used before, why did you change it? Just curious to know

Thanks for the review. I have changed it as I wanted to modify its layout.
